packages/cli/test/integration/webhooks.test.ts

```diff
@@ -21,7 +21,7 @@ describe('WebhookServer', () => {
 
 	describe('CORS', () => {
 		const corsOrigin = 'https://example.com';
-		const activeWebhooks = mockInstance(ActiveWebhooks);
+		const activeWebhooks = mockInstance(LiveWebhooks);
```

Comment:
```suggestion
		const liveWebhooks = mockInstance(LiveWebhooks);
```

Reply:
Nice catch! Completely missed that one 🤦